### PR TITLE
Resolving the text overflow in Divs ( "What comes after implementation" and "About" section)

### LINKED ISSUE #76: Text Overflow in Divs
## Description
Text overflows in boxes under the "WHAT COMES AFTER IMPLEMENTATION" and "ABOUT" heading when viewed on devices with a width between 768px to around 1300px.

## Reproduction
1. Open this [link](https://creativecommons.org/platform/toolkit/).
2. Open the Developer Tools
3. Change the screen width accordingly
4. See the overflow.

## Expectation
There should not be any overflow in these cases too.

## Screenshots
Screenshot from 2021-01-22 22-52-39
Screenshot from 2021-01-22 22-52-52

## Environment
 - Device: laptop
 - OS: Ubuntu 20.04
 - Browser: Chrome, Firefox
 - Version: 88.0.4324

## Resolution
- [ ] I would be interested in resolving this bug.

### PR DESCRIPTION
**Pull request by renuka142**
## Fixes
fixes #76 by @shubhanshu02
## Description
The PR resolves the text overflow occuring in the Div section i.e
 - What comes after implementation
 - About
 
When viewed on devices with a width between 768px to around 1300px.
## Technical Details
changed some hardcoded widths and heights from these cards , for the pt-card and pt-text-card classes:

## Tests
1) Open the "ABOUT" section of the page.
2) Adjust the width size between 768px to 1300px.
3) Notice that the text overflow which was occuring previously is rectified.
4) Follow step 1 and 2 for "WHAT COMES AFTER IMPLEMENTATION" section.

## Screenshot
Screenshot (167)

Screenshot (169)

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

File: docs/index.html [hunk 1]
```diff
@@ -269,7 +269,7 @@ <h3 class="padding-left-small padding-bottom-large">About</h3>
         <div class="pt-card-large">
           <h4>What</h4>
           <h3 class="b-header pt-header-card pt-text-card">Onboarding material for license implementation</h3>
-          <p class="text-normal">This toolkit brings all the elements for a basic Creative Commons platform integration. </p>
+          <p class="text-normal">This toolkit brings all the elements for a basic Creative Commons platform integration.</p>
```
Comment (Cronus1007) [inline docs/index.html:272]: @renuka142 Thanks for your contribution! 
Plzzz revert this unnecessary change of linting. Make it in accordance with the complete repo.

File: docs/src/css/style.css [hunk 2]
```diff
@@ -289,7 +292,8 @@ a.scroll-link:hover {
   }
 }
 .pt-text-card{
-  width: 250px;
+  width: 100%;
+
```
Comment (Cronus1007) [inline docs/src/css/style.css:296]: @renuka142 Unnecessary spacing.

File: docs/src/css/style.css [hunk 3]
```diff
@@ -332,6 +336,7 @@ a.scroll-link:hover {
   #intro>div>div{
     margin: 0 !important;
   }
+
```
Comment (Cronus1007) [inline docs/src/css/style.css:339]: @renuka142 Unnecessary spacing.

File: docs/src/css/style.css [hunk 4]
```diff
@@ -462,3 +467,4 @@ a.scroll-link:hover {
     margin-left: 0;
   }
 }
+
```
Comment (Cronus1007) [inline docs/src/css/style.css:470]: @renuka142 Unnecessary spacing.

File: docs/src/css/style.css [hunk 5]
```diff
@@ -167,6 +168,8 @@ a.scroll-link:hover {
   border-bottom: none;
   background: #ffffff;
   color: #333;
+  
+
```
Comment (Cronus1007) [inline docs/src/css/style.css:172]: @renuka142 Unnecessary spacing.

File: docs/src/css/style.css [hunk 6]
```diff
@@ -119,11 +119,12 @@ a.scroll-link:hover {
   height: 40px;
 }
 .pt-section1-text {
-  width: 250px;
+  width: 100%;
   margin: auto;
 }
 .pt-section-gold {
   border-top: 1px solid #efbe00;
+
```
Comment (Cronus1007) [inline docs/src/css/style.css:127]: @renuka142 Unnecessary spacing.

**Review by Cronus1007 — CHANGES_REQUESTED** [review]

**Comment by Cronus1007** [comment]
@renuka142 The PR seems good. Have a look upon the required changes.

**Comment by renuka142** [comment]
Hello @Cronus1007 , sure i will do the above changes.
Thankyou for your remarks!

**Comment by Cronus1007** [comment]
@renuka142 Have a look about this ss
Screenshot from 2021-02-14 14-58-20
.

**Comment by renuka142** [comment]
Okay , the two cards needs to be aligned.

**Review by brylie — APPROVED** [review]

**Pull request merged**